private/bufpkg/bufcas/digest.go

```diff
@@ -20,11 +20,9 @@ import (
 	"errors"
 	"fmt"
 	"io"
-	"sort"
```

Comment:
Merged all changes from bufmod to this package (except for the new syserror package). I can merge that as well if we want to keep them in sync but didn't want to pull more than was needed.